pkg/appfile/helm/schema.go

```diff
@@ -209,6 +209,7 @@ func handleItemsOfArrayType(t map[string]interface{}) {
 	if t["type"] == "array" {
 		if i, ok := t["items"].([]interface{}); ok {
 			itemSpec, _ := i[0].(map[string]interface{})
```

Comment:
1. The array could be empty, check the length first.

2. The convert can be different types, you should write a switch for it.
  ```
  switch i[0].(type){
   case map[string]interface{}: 
   case []interface{}:
   defaut:  // other types..
   ...
  }
  ```